client/web/src/components/popovers/ColorPicker.vue

```diff
@@ -46,7 +46,7 @@
 	}
 
 	.opacity-picker {
-		background: linear-gradient(to bottom, #ff0000, transparent);
+		background: linear-gradient(to bottom, var(--hue), transparent);
```

Comment:
We want the final color here, not the hue.

![](https://files.keavon.com/-/SlimySlategrayRedhead/capture.png)